app/policies/store_policy.rb

```diff
@@ -7,9 +7,9 @@ class StoresPolicy < ApplicationPolicy
 
   class Scope < ApplicationPolicy::Scope
     # NOTE: Be explicit about which records you allow access to!
-    # def resolve
-    #   scope.all
-    # end
+    def resolve
+      scope.all
+    end
   end
 
   def results?
```

Comment:
You can also remove the `results?` method since you've declared `scope.all` above.